compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/selection/TextFieldSelectionManager.kt

```diff
@@ -612,22 +612,6 @@ internal class TextFieldSelectionManager(
         )
     }
 
-    internal fun doDoubleTapSelection(touchPointOffset: Offset) {
-        if (value.text.isEmpty()) return
-        enterSelectionMode()
-        state?.layoutResult?.let { layoutResult ->
-            val offset = layoutResult.getOffsetForPosition(touchPointOffset)
-            updateSelection(
-                value = value,
-                transformedStartOffset = offset,
-                transformedEndOffset = offset,
-                isStartHandle = false,
-                adjustment = SelectionAdjustment.Word
-            )
-            dragBeginOffsetInText = offset
```

Comment:
`dragBeginOffsetInText = offset` - I think this line is redundant for double tap selection